PxWeb/Code/Api2/Serialization/Csv2DataSerializer.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Http;
+using PCAxis.Paxiom;
+
+namespace PxWeb.Code.Api2.Serialization
+{
+    public class Csv2DataSerializer : IDataSerializer
+    {
+        public void Serialize(PXModel model, HttpResponse response)
+        {
+            response.ContentType = "text/csv; charset=" + System.Text.Encoding.Default.WebName;
```

Comment:
You should set charset according to how its done in the serializers. For example see link: https://github.com/statisticssweden/PCAxis.Core/blob/d1035b8dbe7f897544e4d5df40844f4fb96d3e45/PCAxis.Core/Serializers/CsvFileSerializer.vb#L42